src/applications/disability-benefits/526EZ/tests/config/documentUpload.unit.spec.jsx

```diff
@@ -8,11 +8,13 @@ import { DefinitionTester, // selectCheckbox
 import formConfig from '../../config/form.js';
 import initialData from '../schema/initialData.js';
 
+// NOTE: This file probably duplicates recordUpload
```

Comment:
Looks like they're two separate pages:
- `documentUpload` is for lay statements and other uploads
- `recordUpload` is for private medical records

I definitely had to look that up, though. Could probably use a rename.

Reply:
Yeah, I finally came to the same conclusion as well. I think I'll rename them in a separate PR.